### PR TITLE
init base project

### PR DESCRIPTION
**Pull request by duongngocthach**
init project
link: https://edu-redmine.sun-asterisk.vn/issues/9758

### CODE REVIEW

**Comment by duongngocthach** [comment]
ready

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.example.youtube_background_03"
```
Comment (tuanlvt-1117) [inline app/build.gradle]:
```suggestion
        applicationId "com.sun.youtube_background_03"
```

File: app/src/androidTest/java/com/example/youtube_background_03/ExampleInstrumentedTest.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+    @Test
+    public void useAppContext() {
+        // Context of the app under test.
+        Context appContext = InstrumentationRegistry.getTargetContext();
+
+        assertEquals("com.example.youtube_background_03", appContext.getPackageName());
```
Comment (tuanlvt-1117) [inline app/src/androidTest/java/com/example/youtube_background_03/ExampleInstrumentedTest.java]:
```suggestion
        assertEquals("com.sun.youtube_background_03", appContext.getPackageName());
```

File: app/src/test/java/com/example/youtube_background_03/ExampleUnitTest.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+    @Test
+    public void addition_isCorrect() {
+        assertEquals(4, 2 + 2);
+    }
+}
```
Comment (tuanlvt-1117) [inline app/src/test/java/com/example/youtube_background_03/ExampleUnitTest.java]: enter end line

**Comment by duongngocthach** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**